interface1.html

```diff
@@ -1312,6 +1312,16 @@ <h3>...counting leaves</h3>
 					</div>
 
 					<div onclick="showMore('moreLayeringConstraints')" class="show-more">Show more...</div>
+					<div id="moreLayeringConstraints" class="more-constraints">
```

Comment:
This constraint should've been added to the existing "show more" section below. As it is, the remaining "show more" constraints just don't seem to be available.